UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeCalibrationWarning.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.Events;
+using Microsoft.MixedReality.Toolkit.Input;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+	/// <summary>
+	/// Checks whether eyes are calibrated and prompts a notification to encourage the user to calibrate.
+	/// </summary>
+	[AddComponentMenu("MRTK/Examples/Eye Calibration Notifier")]
+	public class EyeCalibrationWarning : MonoBehaviour
+	{
+        [Tooltip("The EyeCalibrationChecker used to notify the user about calibration status.")]
```

Comment:
Nit ... fix indentation 

Probably can easily use Visual Studio to fix document formatting (CTRL+K CTRL+D)

![image](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/36461279/8d53deb9-4287-4273-8353-b111112cc0b7)